tsconfig.json

```diff
@@ -1,17 +1,16 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "target": "es2019",
+    "esModuleInterop": true,
+    "target": "ES2019",
     "lib": [
-      "es2019",
-      "esnext.weakref",
-      "dom",
-      "dom.iterable"
+      "ESNext",
+      "DOM",
+      "DOM.Iterable"
     ],
-    "module": "commonjs",
-    "outDir": "./dist",
-    "strict": true,
-    "esModuleInterop": true
+    "module": "CommonJS",
+    "outDir": "dist",
+    "strict": true
   },
-  "exclude": ["index.d.ts"]
+  "exclude": ["index.d.ts", "main", "renderer"]
```

Comment:
can you undo the non-semantic changes here (case changes, reordering, changing `./dist` to `dist`)? Thanks :)

Reply:
Done :)